template-helpers/dashboard.js

```diff
@@ -2,6 +2,19 @@
 
 const { LocaleUtils } = require("./../locale-utils");
 const { makeBreachCards } = require("./breaches");
+const { hasUserSignedUpForRelay } = require("./../controllers/utils");
+
+function showRelayCallout(args) {
+  const enLocaleIsSupported = args.data.root.req.supportedLocales.includes("en");
```

Comment:
`supportedLocales` always includes `en` because we specify it as our `defaultLocale`. This check needs to specifically check the `args.data.root.req.headers["accept-language"]` instead.

```suggestion
  const enLocaleIsSupported = args.data.root.req.headers["accept-language"].includes("en");
```